src/components/Navigation/NavigationItem/_navigationItem.scss

```diff
@@ -69,6 +69,10 @@
     margin-bottom: 0;
   }
 
+  a {
+    display: inline;
```

Comment:
This should probably be inline-block. Unless  this is to shrink the clickable area

Reply:
Looking on mobile, I think we may want to add a media query to only apply this to desktop sizes. I think on mobile the expectation would be to have the whole menu bar be clickable, no?

Reply:
@constantinehuzenko Was this added explicitly for mobile? I think the expectation on mobile would be that the whole line would be the link and not jsut the texts, because of how we have styled it. 

On desktop I would make it inline-block. Also the LI seems to be `inline`. That should be made to be inline-block too or removed all together .